streaming/text/convert/laion/laion400m/download_data.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+wandb login
```

Comment:
is this mandatory? what if user doesn't have a wandb account? can we just comment this line and add a comment saying `uncomment this line to see the metric in wandb` ?

Reply:
Updated